### PR TITLE
download helper / setup for Windows

### PR DESCRIPTION
**Pull request by ninjamuffin99**
written for the unstable branch 

redid some of the commandline stuff, to sort out auto downloading of the dependencies and whatnot. nothing too robust, but should be work that can be generally easily expanded upon and worked with

replaces the homemade cli tools with tink_cli, and then wrote a few scripts (stole from google search lol!) to download the dependencies for WINDOWS. specific work will need to be done for mac and linux!

TLDR: `haxelib run hxcodec setup windows` basically

### CODE REVIEW

**Comment by ninjamuffin99** [comment]
converted to draft, since `tink_cli` doesn't work properly with haxe 4.3

**Comment by polybiusproxy** [comment]
nice! if you need any help with anything, don't hesitate to tell me 👍

**Comment by ninjamuffin99** [comment]
thanks! if someone were to get a setup for Mac or Linux that would be pleasant, although I could probably whip that up myself at some point for Mac at least

**Comment by polybiusproxy** [comment]
> thanks! if someone were to get a setup for Mac or Linux that would be pleasant, although I could probably whip that up myself at some point for Mac at least

I haven't got any Mac/Linux machine. ☹️
I could perhaps whip up WSL, and use QEMU (inside WSL) for macOS.

**Comment by EliteMasterEric** [comment]
Closed as this is no longer necessary (LibVLC 4.0 is no longer being used so this tool isn't necessary anymore), sorry Cam😛
